src/components/tooltips/APRTooltip/APRTooltip.spec.ts

```diff
@@ -124,5 +128,35 @@ describe('APRTooltip', () => {
     });
   });
 
-  // describe('Token Aprs', () => {});
+  describe('Token Aprs', () => {
+    it('Should show stETH staking reward APRs', () => {
+      const aprBreakdown: AprBreakdown = {
+        ...EmptyAprBreakdownMock,
+        tokenAprs: {
+          total: 166,
+          breakdown: {
+            '0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0': 166,
+          },
+        },
+        min: 166,
+        max: 166,
+      };
+      const poolMock: Pool = {
+        ...EmptyPoolMock,
+        tokensList: [configService.network.addresses.wstETH],
+      };
+      const { container } = render(APRTooltip, {
+        props: {
+          pool: poolMock,
+          poolApr: aprBreakdown,
+        },
+      });
+      expect(container.getElementsByClassName('total-apr')[0].textContent).toBe(
```

Comment:
These tests are great but I would like to start a little discussion:

[Testing library](https://testing-library.com/docs/queries/about#priority) recommends to use accesible or semantic queries instead of CSS classnames. 

At the same time, in this concrete case, using `getByRole` or `getByText` would not be so specific as `getElementsByClassName('total-apr')` that you used here. 

Because of that, in this case I would use `data-testid` attribute in the template like this: 

```html
 <div
        data-testid="total-apr"
        class="px-3 pt-3 pb-1 bg-gray-50 dark:bg-gray-800 rounded-t total-apr"
 >
```
so that we can replace the expects block by this simpler to read and less brittle version:

```js
expect(screen.getByTestId('total-apr').textContent).toBe(
        'Total APR15.22%'
);

expect(screen.getByTestId('swap-fee-apr').textContent).toBe(
      '15.22% Swap fees APR'
);
```
I agree that it would involve adding a bit of pollution to the html ([related discussion here](https://github.com/testing-library/react-testing-library/issues/683)) just for testing purposes but, I think that it is worthy given the tradeoffs. 

What do you think? Did you already had this conversation in the team?

Reply:
Good idea, implemented.